reactivexcomponent/tests/unit/mock_publisher_dependencies.py

```diff
@@ -0,0 +1,85 @@
+import json
+from unittest.mock import MagicMock
+
+COMPONENT_CODE = -725052640
+STATE_MACHINE_CODE = -2027871621
+EVENT_CODE = 8
+MESSAGE_TYPE = "XComponent.Devinette.UserObject.CheckWord"
+ROUTING_KEY = "input.1_0.microservice1.Devinette.DevinetteChecker"
+GUI_EXAMPLE = "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx"
```

Comment:
not a valid GUID, use a random fixed guid instead of a fake invalid one
e.g

`C56A4180-65AA-42EC-A945-5FD21DEC0538`

rename the variable to simply `GUID`, all these variables are examples anyway.